app/src/main/java/com/bruno/aybar/composechallenges/flappy/FlappyBirdScreen.kt

```diff
@@ -60,11 +57,15 @@ fun GameArea(viewModel: FlappyBirdViewModel, modifier: Modifier) {
 
     val state = viewModel.state.observeAsState(FlappyGameUi.NotStarted)
 
-    WithConstraints(modifier) {
-        onActive {
+    BoxWithConstraints(modifier) {
+        LaunchedEffect(viewModel) {
             viewModel.onGameBoundsSet(maxWidth.value, maxHeight.value)
         }
-        Box(Modifier.clickable(onClick = viewModel::onTap, indication = null)) {
+
+        Box(Modifier
+            .clickable(onClick = viewModel::onTap, indication = null, interactionState = remember { InteractionState() })
+            .fillMaxSize()
```

Comment:
`.fillMaxSize()` is required now, because this Box is now a child of BoxWithConstraints. If we don't use explicitly ask it to occupy all the space, it won't do it (that was not the behavior with `WithConstraints`)